geoips/plugins/modules/readers/ami_netcdf.py

```diff
@@ -106,9 +107,7 @@
 }
 
 
-DONT_AUTOGEN_GEOLOCATION = False
-if os.getenv("DONT_AUTOGEN_GEOLOCATION"):
-    DONT_AUTOGEN_GEOLOCATION = True
+DONT_AUTOGEN_GEOLOCATION = PATHS["DONT_AUTOGEN_GEOLOCATION"]
```

Comment:
Is this the same behavior as prior? The previous behavior set this to True no matter what the value of "DONT_AUTOGEN_GEOLOCATION" was, as long as it was set.

Reply:
Please see my previous comments.